executor/src/main/java/org/k8loud/executor/action/openstack/AttachVolumeAction.java

```diff
@@ -0,0 +1,30 @@
+package org.k8loud.executor.action.openstack;
+
+import data.Params;
+import org.k8loud.executor.exception.ActionException;
+import org.k8loud.executor.exception.OpenstackException;
+import org.k8loud.executor.openstack.OpenstackService;
+
+public class AttachVolumeAction extends OpenstackAction {
+    private String region;
+    private String serverId;
+    private String volumeId;
+    private String device;
+
+    public AttachVolumeAction(Params params, OpenstackService openstackService) throws ActionException {
+        super(params, openstackService);
+    }
+
+    @Override
+    public void unpackParams(Params params) {
+        region = params.getRequiredParam("region");
+        serverId = params.getRequiredParam("serverId");
+        volumeId = params.getRequiredParam("volumeId");
```

Comment:
Maybe move common fields declaration and initialization to a `VolumeAction`  or `AttachDetachVolumeAction` class and extend it?